lib/routes/asianfanfics/tag.ts

```diff
@@ -0,0 +1,83 @@
+import { DataItem, Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { parseDate } from '@/utils/parse-date';
+import { load } from 'cheerio';
+
+// test url http://localhost:1200/asianfanfics/tag/milklove/N
+
+export const route: Route = {
+    path: '/tag/:tag/:type',
+    categories: ['reading'],
+    example: '/asianfanfics/tag/milklove/N',
+    parameters: {
+        tag: '标签',
+        type: '排序类型',
+    },
+    name: '亚洲同人网标签',
+    maintainers: ['KazooTTT'],
+    radar: [
+        {
+            source: ['www.asianfanfics.com/browse/tag/:tag/:type'],
+            target: '/tag/:tag/:type',
+        },
+    ],
+    description: `匹配亚洲同人网标签，支持类型：
+- L: Latest 最近更新
+- N: Newest 最近发布
+- O: Oldest 最早发布
+- C: Completed 已完成
+- OS: One Shots 短篇
+`,
+    handler,
+};
+
+type Type = 'L' | 'N' | 'O' | 'C' | 'OS';
+
+const typeToText = {
+    L: '最近更新',
+    N: '最近发布',
+    O: '最早发布',
+    C: '已完成',
+    OS: '短篇',
+};
+
+async function handler(ctx) {
+    const tag = ctx.req.param('tag');
+    const type = ctx.req.param('type') as Type;
+
+    if (!type || !['L', 'N', 'O', 'C', 'OS'].includes(type)) {
+        throw new Error('Invalid type');
+    }
+    const link = `https://www.asianfanfics.com/browse/tag/${tag}/${type}`;
+
+    const response = await ofetch(link);
```

Comment:
You have to [pass RSSHub's UA to ofetch header](https://github.com/unjs/ofetch#-adding-headers).